earth2mip/inference_medium_range.py

```diff
@@ -216,8 +189,24 @@ def score_deterministic(
             Used for ACC.
 
     Returns:
-        metrics
-
+        metrics::
```

Comment:
Isnt this of type xarray dataset

Reply:
This output will look like ds.info() returns and will be familiar to anyone who uses ncdump. It's called CDL a plain text data format isomorphic to netCDF.

Reply:
i updated the docstring